flashcards/backend/docs/conftest.rst

```diff
@@ -0,0 +1,7 @@
+conftest module
```

Comment:
I would remove rat for conftest for now

Reply:
Done.

Reply:
Can't resolve the conversation until there are chacnges in remote repo 👯